src/axom/sidre/tests/sidre_group.cpp

```diff
@@ -571,6 +574,55 @@ TEST(sidre_group, child_lists)
   delete ds;
 }
 
+//------------------------------------------------------------------------------
+// Test Group's list format for holding contents of a vector of strings.
+//------------------------------------------------------------------------------
+TEST(sidre_group, string_list)
+{
+  // Round-trip test from std::vector<std::string> to Group and back.
+  DataStore* ds = new DataStore();
+  Group* root = ds->getRoot();
+
+  std::vector<std::string> str_vec;
+  str_vec.push_back("This");
+  str_vec.push_back("is");
+  str_vec.push_back("a");
+  str_vec.push_back("vector");
+  str_vec.push_back("to");
+  str_vec.push_back("test");
+  str_vec.push_back("strings");
+  str_vec.push_back("in");
+  str_vec.push_back("sidre::Group's");
+  str_vec.push_back("list");
+  str_vec.push_back("format");
+
+  // my_strings is a Group in list format.
+  Group* my_strings = root->createGroup("my_strings", true);
```

Comment:
Thanks @nselliott 

I think the boolean param is the important part, right?
If so, can you please make the following (small) change?
```suggestion
  const bool use_list_collection = true;
  Group* my_strings = root->createGroup("my_strings", use_list_collection);
```


See: https://github.com/LLNL/axom/blob/cab5877121dbc467a716237eef1128c861a4baba/src/axom/sidre/core/Group.hpp#L934-L946